tests/unit/test_settings.py

```diff
@@ -0,0 +1,49 @@
+#  Copyright 2024 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from unittest.mock import patch
+
+import pytest
+
+from aap_eda.settings.default import (
+    ImproperlyConfigured,
+    RulebookProcessLogLevel,
+    _get_rulebook_process_log_level,
```

Comment:
@Alex-Izquierdo Since  _get_rulebook_process_log_level is now being shared across modules can we remove the _ prefix, which means its local.

Reply:
I disagree. The scope of _get_rulebook_process_log_level is limited only to the defaults.py. A test file can not be considered a public usage. Private methods/functions can (and should) be tested too.